### DISCUSSION TITLE
Custom css in doks-child-theme

### DISCUSSION THREAD

**Discussion by vivekragunathan** [thuliteio/doks#591] [Q&A] [answered]
In the README ...

> _Doks is available as a child theme, and a starter theme:_
>   _Use the Doks child theme, if you do not plan to customize a lot, and/or need future Doks updates._
>   _Use the Doks starter theme, if you plan to customize a lot, and/or do not need future Doks updates._

I need to add a few lines of custom css to my docs. I don't want to go with the doks `starter` theme and give up getting future Doks updates (and will not be customizing a lot). In Hugo, you do this by adding `customCss` field in the `config.toml`. Is there a way in `child-theme` to include a custom css/scss file as part of the build so that I can add my custom style to that file?

I appreciate any advice in this regard.

**Comment by h-enk** [comment, marked as answer]
See: https://github.com/h-enk/doks/discussions/577#discussioncomment-1840823

I will make this part of the upcoming release (working on it) — see also: https://github.com/h-enk/doks-child-theme/issues/99

**Reply by vivekragunathan** [reply]
You are awesome, man. Thanks.

**Reply by assayire** [reply]
Is the ability to provide custom css available in the latest version (`0.4.0`) of doks? Specifically, is it available in the doks child theme?

Reasons I ask:

- Issue #591 points to #655, which shows as still open
- I like to stick to child theme but looks like the change suggested [here](https://github.com/h-enk/doks/issues/655#issuecomment-1010396591) in #655 would make the child theme diverge from its source.

**Reply by h-enk** [reply]
Planned for [v0.4.1](https://github.com/h-enk/doks/milestone/6), planned dropping end of this week /beginning of next week. Will be made available in both the starter and child theme.
